### PR TITLE
Create environment route

### LINKED ISSUE #42: Create a route sends environment to frontend

### PR DESCRIPTION
**Pull request by joshcwinton**
Description of PR here...

Issue: Closes #42 

### Frontend:
- 
### Backend:
- create `getEnvironment` function in `environment.py` that returns GRID
- create get route called `/environment` that sends grid array in json 
- create get route called `/environment_image` that serves static image file

### CODE REVIEW

File: backend/environment.py [hunk 1]
```diff
@@ -64,8 +73,14 @@ def clearBoard():
...
+    
+    renderEnvironment(GRID)
 
 def showGrid():
     for i in GRID:
         print(i)
 
+    renderEnvironment(GRID)
+
+def getEnvironment():
```
Comment (JasperCheung) [inline backend/environment.py:85]: maybe wanna use this in chatbox instead of just import GRID.

Comment (joshcwinton) [reply backend/environment.py:85]: do you think that's big enough for a new issue

Comment (JasperCheung) [reply backend/environment.py:85]: sure, lets document it. I don't think the change has to happen now? Maybe make a issue and add a `TODO: change this to getEnviroement` somewhere in `enviroment.py`

Comment (joshcwinton) [reply backend/environment.py:85]: added todo and #53

**Pull request merged**